service/create/service.go

```diff
@@ -447,6 +447,7 @@ func (s *Service) runMachine(input runMachineInput) error {
 		return microerror.MaskAny(err)
 	}
 
+	az := input.spec.AWS.Region + input.spec.AWS.AZ
```

Comment:
Yes, the AZ identifier is of the form "eu-west-1a", cfr. [this](https://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html#examples) (look for the `Placement` section).

Is your proposal that we use Amazon's format in the TPR?